docs/src/data/hid.js

```diff
@@ -7865,4 +7865,25 @@ export default [
     },
     footnotes: {},
   },
+  {
+    names: ["C_AC_NEXT_KEYBOARD_LAYOUT_SELECT", "GLOBE"],
+    description: "AC Next Keyboard Layout Select (Apple Globe)",
+    context: "Consumer AC",
+    clarify: true,
+    usages: [
+      {
+        application: consumerApplication,
+        item: usage(consumerPage, 0x29d),
+      },
+    ],
+    documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=150",
```

Comment:
```suggestion
    documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=153",
```

Apologies if this is just browser-pdf-nonsense, but for me the usage appears on `#page=153` (marked "152" in the PDF). This URL seems to take me to the correct page in Firefox and Chrome, at least.